ckanext/switzerland/helpers/dataset_form_helpers.py

```diff
@@ -281,20 +287,8 @@ def ogdch_temporals_form_helper(data):
 
 
 def _get_temporals_from_storage(data, key):
-    """
-    data is expected to be stored as:
-    "temporals": [{"start_date": "1981-06-14T00:00:00",
-     "end_date": "2020-09-27T00:00:00"}]
-    """
     if data:
-        temporals = data.get(key)
```

Comment:
Do temporals no longer need special treatment? That is great, just wanted to make sure it is handled elsewhere or can just be ignored 😃  (in case it got forgotten)

Reply:
Here's how it works now: temporals, the modified date and the issued date are all stored in the db as posix timestamps. The datepicker uses dd.mm.yyyy format (in all four languages). There's a validator to convert them to posix on saving, and an output validator to convert them to dd.mm.yyyy on fetching them, which has already happened at this point in the code, so no need to convert them. (We use both those validators in both the `ogdch_date` and `temporals` presets.)

When we display the dates on the dataset, we convert them again from dd.mm.yyyy to a text format (like `1. Dezember 2020`). There's a new frontend helper function for that. I'm open to changing this around so that the output validator converts to text, and then we convert from that to dd.mm.yyyy for the datepicker on the edit page, if you think it's more efficient that way... but I thought this was simpler and more logical to understand.

The old default datepicker could only handle dates in isodate format, which is why there was stuff about isodate in the code, and I think for a while we were saving temporals in isodate in the db and modified/issued dates in posix still... I have tried to clean all that mess up. There's still one or two places in the code where we *can* handle isodate if necessary, because I wanted to be on the safe side, but we could probably get rid of those too.

Hope that explains it!

Reply:
Thank you so much for the explanation (I just wrote a comment, but it is apparently not posted, when I click resolve :P ) 
I don't see why we should change this implementation, it is readable and makes sense to me.
So I am happy with your solution if you are, too :)